main.go

```diff
@@ -149,10 +152,13 @@ func main() {
 		// remove trailing /
 		apiPrefix = strings.TrimRight(apiPrefix, "/")
 	}
+
+	appNameMap := configureAppNameMap(cfg)
+
 	log.Printf("Using %s/listing as public URI", apiPrefix)
 
 	rand.Seed(time.Now().UnixNano())
-	http.Handle("/api/submit", &submitServer{ghClient, glClient, apiPrefix, slack, genericWebhookClient, cfg})
+	http.Handle("/api/submit", &submitServer{ghClient, glClient, apiPrefix, slack, genericWebhookClient, appNameMap, cfg})
```

Comment:
> - i don't think we need to make a function for just this part of the problem; other than to factor it out of the main method (which is actually one big NewSubmitServer function).

I mean yes, there's a whole lot of stuff that could do with factoring out to a function that just creates the SubmitServer without calling `http.Handle` etc. But it's fine to leave this for now.